mycli/main.py

```diff
@@ -1160,7 +1160,7 @@ def cli(database, user, host, port, socket, password, dbname,
 
         try:
             sys.stdin = open('/dev/tty')
-        except FileNotFoundError:
+        except (FileNotFoundError, OSError):
```

Comment:
So, the way I originally implemented this probably wasn't very good. Here `FileNotFoundError` might [actually be `OSError`](https://github.com/dbcli/mycli/blob/129b579f0c2766f69abe3a9058db264b9a7480af/mycli/main.py#L53:L59). That's even more confusing in that `FileNotFoundError` is a subclass of `OSError` in Python 3.

How about we just catch `IOError` (Python2) and `OSError` (Python3)? And, get rid of the weird import thing I did with `FileNotFoundError`?